web-registry/src/components/organisms/BasketOverview.tsx

```diff
@@ -0,0 +1,274 @@
+  {
+    id: 'curator',
+    displayName: 'curator',
+    value: 'Regen Network Development, Inc',
+    link: 'https://www.regen.network/',
+  },
+  {
+    id: 'allowedCreditClasses',
+    displayName: 'allowed credit classes',
+    value: 'VCU (CO2)',
+    link: 'https://www.regen.network/',
+  },
+  {
+    id: 'minStartDate',
+    displayName: 'min start date',
+    value: '10-year rolling acceptance window',
+  },
+];
+
+// TODO: Basket description (see comment in Figma)
+// https://www.figma.com/file/x5vjWsddiUBzP2N13AFOPw?node-id=32:10028#155844414
+const basketDescription =
+  'The Regen Nature Carbon Ton groups together carbon sequestration ecocredits into one tradeable asset. Lorem ipsum dolor sit amet, consectetur adipiscing elit.';
+
+interface BasketOverviewProps {
+  basketDenom: string;
+}
+
+const BasketOverview: React.FC<BasketOverviewProps> = ({ basketDenom }) => {
+  const styles = useStyles();
+  const [basket, setBasket] = useState<Basket>();
+
+  useEffect(() => {
+    if (!basketDenom || !ledgerRestUri) return;
+
+    const fetchData = async (): Promise<void> => {
+      const _basket = await getBasket(basketDenom);
+      setBasket(_basket);
+    };
+
+    fetchData();
+  }, [basketDenom]);
+
+  return (
+    <Box className={styles.top}>
+      <Section className={styles.content}>
+        <Grid container>
+          <Grid item xs={12} sm={5} className={styles.imageContainer}>
+            <OptimizedImage
+              className={styles.image}
+              src={forestImg}
+              alt={basket?.name}
+            />
+          </Grid>
+          <Grid item xs={12} sm={7} className={styles.textContainer}>
+            <Title variant="h1" className={styles.title}>
+              {basket?.name}
+            </Title>
+            <Description className={styles.basketDenom}>
+              {basket?.basket_denom}
+            </Description>
+            <Description className={styles.basketDescription}>
+              {/* TODO: Hardcoded description */}
+              {basketDescription}
+            </Description>
+            <OnBoardingCard className={styles.card}>
+              <Grid
+                container
+                rowSpacing={1}
+                columnSpacing={{ xs: 1, sm: 2, md: 3 }}
+              >
+                {basketSummary.map(item => (
+                  <Grid item key={item.id} xs={12} sm={6}>
+                    <Item
+                      label={item.displayName}
+                      data={item.value}
+                      link={item.link}
+                    />
+                  </Grid>
+                ))}
+              </Grid>
+            </OnBoardingCard>
+          </Grid>
+        </Grid>
+      </Section>
+    </Box>
+  );
+};
+
+export { BasketOverview };
+
+/**
+ * Basket summary item (subcomponent)
+ */
+
+const useStylesItem = makeStyles(theme => ({
+  gridItem: {
+    marginTop: theme.spacing(4),
+  },
```

Comment:
this is maybe better for an engineering call (or github discussion like we discussed previously), but I think we might want to avoid using `makeStyles` as much as possible moving forward, as it won't work in React 18 (see the disclaimer at the top of [this page](https://mui.com/styles/basics/)

For a bit of context, we were on MUI v3 until recently (skipped 4, went straight to 5) and with MUI v3, `makeStyles` was the only styling option. It uses JSS under the hood, but MUI switched to Emotion with v5 and JSS is now deprecated. 

I've been playing with the options some, and my personal preference is using the [sx prop](https://mui.com/system/the-sx-prop/#main-content) for positioning (padding, margin etc) and MUI's [styled](https://mui.com/customization/how-to-customize/#2-reusable-style-overrides) for reusable styles. 

There will also be places where it won't make sense to use the new options (mainly when interacting with older components which accept `classes` or a JSS `className` as a prop you need to pass - as you've done several places in this story), and It might be annoying to have 3 forms of styles in the same codebase (`styled`, `makeStyles`, and `cx`), but they're all connected and assuming MUI doesn't change their plans moving forward, we'll eventually have to migrate away from `makeStyles` to upgrade react, and using the newer syntax now might help with that.

Again, this is probably not the best place to discuss, but I wanted to mention given your recent push for code quality maintenance.

Reply:
I take note, thanks a lot!